mods/bones/init.lua

```diff
@@ -190,21 +190,29 @@ minetest.register_on_dieplayer(function(player)
 		end
 	end
 
+	local invlist = {}
+	local droplist = {}
+
 	if bones_mode == "drop" then
 
 		-- drop inventory items
 		for i = 1, player_inv:get_size("main") do
+			local stackname = player_inv:get_stack("main", i):to_string()
+			if stackname ~= "" then droplist[#droplist+1] = stackname end
 			drop(pos, player_inv:get_stack("main", i))
 		end
 		player_inv:set_list("main", {})
 
 		-- drop crafting grid items
 		for i = 1, player_inv:get_size("craft") do
+			local stackname = player_inv:get_stack("main", i):to_string()
+			if stackname ~= "" then droplist[#droplist+1] = stackname end
 			drop(pos, player_inv:get_stack("craft", i))
 		end
 		player_inv:set_list("craft", {})
 
 		drop(pos, ItemStack("bones:bones"))
+		minetest.log("action","Dropped "..player_name.."'s items: "..table.concat(droplist,", "))
```

Comment:
Writing the items in question isn't needed IMO, the log output for chests doesn't mention the items either